lib/dropbox_api/metadata/file.rb

```diff
@@ -26,6 +26,7 @@ class File < Base
     field :rev, String
     field :size, Integer
     field :content_hash, String, :optional
+    field :media_info, Hash, :optional
```

Comment:
The `media_info` field's class is actually `MediaInfo`. We should implement this type instead of wrapping all values in a Hash.

One practical benefit of doing so is that we'd free the library user from the hassle of type casting, which is a priority goal for this API wrapper. Ie. you would need to do something like `Date.parse(file.media_info["metadata"]["photo"]["time_taken"])`.

If we implemented a MediaInfo class, the value of `file.media_info.metadata.photo.time_taken` would be a `Date` straight away.

Reply:
@Jesus I am agree with you. Will do it soon.